deepchem/models/torch_models/infograph.py

```diff
@@ -217,16 +445,16 @@ def unsup_sup_loss(self, inputs, labels, weights):
         loss = self._global_global_loss(g_enc, g_enc1)
         return loss
 
-    def _local_global_loss(self, l_enc, g_enc, batch):
+    def _local_global_loss(self, l_enc, g_enc, index):
```

Comment:
This should be factored out to be a loss function (we can remove the self.device, self.measure, self.average_loss calls by making appropriate flags). This is a general mathematical loss term that might be useful in different contexts